torch/_dynamo/variables/functions.py

```diff
@@ -646,6 +646,21 @@ def get_val(v):
                 **{k: get_val(v) for k, v in self.keywords.items()},
             )
 
+    def reconstruct(self, codegen):
+        codegen.load_import_from("functools", "partial")
+        codegen(self.func)
+        if self.args and self.keywords:
+            # TODO(voz): Is there a util to help with this? neither the existing ones suffice?
+            unimplemented("Reconstruction of mixed args and keywords NYI")
```

Comment:
@jansel do we need this? godbolt says python unpacks mixed kwargs/args as just args https://godbolt.org/z/5T6eKefrv but I don't know if that's always right...

Reply:
https://docs.python.org/3/library/dis.html#opcode-CALL_FUNCTION_EX